ocs_ci/helpers/tlsprofile_helper.py

```
@@ -0,0 +1,1215 @@
+    Return log lines that match TLS-related error heuristics for a single pod.
+    """
+    from ocs_ci.ocs.resources.pod import get_pod_logs
+
+    try:
+        raw = get_pod_logs(
+            pod_name=pod_name,
+            namespace=namespace,
+            since=since,
+            tail=str(tail),
+        )
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)